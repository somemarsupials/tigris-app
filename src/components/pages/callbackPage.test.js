import React from 'react';
import { shallow } from 'enzyme';
import CallbackPage from './callbackPage';

describe('CallbackPage', () => {
  let props;

  describe('when error', () => {
    beforeEach(() => {
      props = {
        error: {
          status: 500, 
          data: {
            message: 'error'
          },
        },
      };
    });

    it('matches snapshot', () => {
      expect(shallow(<CallbackPage {...props} />)).toMatchSnapshot();
    });
  });

  describe('without error', () => {
    beforeEach(() => {
      props = {};
    });

    it('matches snapshot', () => {
      expect(shallow(<CallbackPage {...props} />)).toMatchSnapshot();
    });
  });
});
