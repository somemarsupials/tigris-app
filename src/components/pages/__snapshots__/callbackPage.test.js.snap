// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CallbackPage matches snapshot 1`] = `
<span>
  Loading info...
</span>
`;
